test/misc.jl

```diff
@@ -16,6 +16,12 @@
 import ConstraintTrees as C
 import SparseArrays as SP
 
+@testset "Bounds" begin
+    b = C.EqualTo(0)
+    @test_deprecated length(b)
+    @test length(b) == 1
```

Comment:
```suggestion
    @test (@test_deprecated length(b)) == 1
```

Doing it like this makes sure the deprecation warning isn't printed in the test logs